.travis.yml

```diff
@@ -1,7 +1,11 @@
 language: rust
 rust:
+  - stable
   - nightly
 script:
-  - cargo test
+  - cargo build
+  - |
+    [ $TRAVIS_RUST_VERSION != nightly ] ||
+    cargo test
```

Comment:
I did try that, but `travis lint` complains:

```
Warnings for .travis.yml:
[x] syntax error: (<unknown>): did not find expected comment or line break while scanning a block scalar at line 7 column 41
```
(column 41 being the start of `||`)